.github/workflows/pr-merged.yml

```diff
@@ -10,20 +10,21 @@ jobs:
         runs-on: ubuntu-latest
         if: github.event.pull_request.merged == true
         steps:
-            - name: Find Asana task ID
-              id: find-task
-              uses: duckduckgo/native-github-asana-sync@v1.5
+            - name: Find Asana task ID(s)
+              id: find-tasks
+              uses: duckduckgo/native-github-asana-sync@c481af86e02402c697e5521278e322ff670d6f44
               with:
-                  action: 'find-asana-task-id'
+                  action: 'find-asana-task-ids'
                   asana-pat: ${{ secrets.ASANA_ACCESS_TOKEN }}
                   trigger-phrase: 'Task/Issue URL:'
+                  asana-project: '1203268166580279' # CPM (Cookie Pop-up Management) Triage project
```

Comment:
Should we [still add](https://app.asana.com/0/0/1209805344481173/1209820083344133/f) any mentioned tasks to the project? Not a strong opinion, just a suggestion.